.github/CHANGELOG.md

```diff
@@ -0,0 +1,61 @@
+## Release 0.6.0a3 (development release)
+
+### New features since the last release
+
+### Bug fixes
+
+### Housekeeping changes
+
+* A large number of linting corrections were made to improve the overall pylint report. These were mostly minor but essential modifications including restructuring, re-coding, optimization, updating `.pylintrc`, adding `.coveragerc`. The code quality score is improved to `A` for the released version. Check ["Files changed"](https://github.com/XanaduAI/flamingpy/pull/11/files) for details.
+* `TODO` comments have been removed exclusively from files with low code quality grades. The Dev team has created tickets to be actioned for all removed TODO comments on separate (private) FlamingPy boards.
+* `CONTRIBUTING.md`, `code_of_conduct.md`, and `CHNAGLELOG.md` were imported and modified from the StrawberryFields project. Dev team plans to extend these with customized details in future PRs.
+
+### Documentation changes
+
+### Contributors
+
+This release contains contributions from (in alphabetical order) ... See full commit details ...
+
+---
+
+## Release 0.4.9a1 (current release)
+
+<h3>Housekeeping changes since the last release</h3>
```

Comment:
```suggestion
### Improvements
```

Reply:
I suggest it's useful to always say "since the last release" for the top one (as in changing "### Improvements since the last release" or if we have another title on top adding "since the last release") -- this way it is clear what we are comparing against even if there are no new features.